src/rqg.scss

```diff
@@ -1043,3 +1047,187 @@ body {
       0 0 0.2em var(--rqg-color-main-bg);
   }
 }
+
+// Tabbed Navigation for Item Sheets
+
+.tabs {
```

Comment:
Maybe split the styling for item sheets into it's own scss file. This file should really be divided since it's way to big already.

Reply:
Agreed we should do this but I think it should be a separate issue so we organize the whole thing, and remove dead-wood.

Reply:
This has not been addressed.

Reply:
Agreed